packages/@webex/plugin-meetings/test/unit/spec/member/util.js

```diff
@@ -1,53 +1,356 @@
 import {assert} from '@webex/test-helper-chai';
 import MemberUtil from '@webex/plugin-meetings/src/member/util';
+import { ServerRoles } from '../../../../src/member/types';
 
-describe('isHandRaised', () => {
-  it('throws error when there is no participant', () => {
-    assert.throws(() => {
-      MemberUtil.isHandRaised();
-    }, 'Raise hand could not be processed, participant is undefined.');
+describe('plugin-meetings', () => {
+  describe('isHandRaised', () => {
+    it('throws error when there is no participant', () => {
+      assert.throws(() => {
+        MemberUtil.isHandRaised();
+      }, 'Raise hand could not be processed, participant is undefined.');
+    });
+  
+    it('returns false when controls is not there', () => {
+      const participant = {};
+  
+      assert.isFalse(MemberUtil.isHandRaised(participant));
+    });
+  
+    it('returns false when hand is not there in controls', () => {
+      const participant = {
+        controls: {},
+      };
+  
+      assert.isFalse(MemberUtil.isHandRaised(participant));
+    });
+  
+    it('returns true when hand raised is true', () => {
+      const participant = {
+        controls: {
+          hand: {
+            raised: true,
+          },
+        },
+      };
+  
+      assert.isTrue(MemberUtil.isHandRaised(participant));
+    });
+  
+    it('returns false when hand raised is false', () => {
+      const participant = {
+        controls: {
+          hand: {
+            raised: false,
+          },
+        },
+      };
+  
+      assert.isFalse(MemberUtil.isHandRaised(participant));
+    });
   });
 
-  it('returns false when controls is not there', () => {
-    const participant = {};
+  describe('MemberUtil.extractControlRoles', () => {
+    it('happy path extract control roles', () => {
+      const participant = {
+        controls: {
+          role: {
+            roles: [
+              {type: 'PRESENTER', hasRole: true},
+              {type: 'COHOST', hasRole: true},
+              {type: 'MODERATOR', hasRole: true},
+            ]
+          }
+        }
+      }
 
-    assert.isFalse(MemberUtil.isHandRaised(participant));
+      assert.deepEqual(MemberUtil.extractControlRoles(participant), {cohost: true, moderator: true, presenter: true});
+    });
   });
 
-  it('returns false when hand is not there in controls', () => {
-    const participant = {
-      controls: {},
-    };
+  describe('MemberUtil.getControlsRoles', () => {
+    it('getControlsRoles', () => {
```

Comment:
nit: the text of an `it` should make a sentence starting with "it"
```suggestion
    it('returns expected result from getControlsRoles', () => {
```